src/Dropdown/Dropdown.js

```diff
@@ -104,6 +106,10 @@ class Dropdown extends Util.mixin(BindMixin) {
   handleMenuToggle(e) {
     e.stopPropagation();
 
+    if (this.props.disabled) {
```

Comment:
🤔  What happens if the dropdown was disabled while it's open?

Reply:
I can ask the same question to the former implementation :trollface: